### DISCUSSION TITLE
```minimum_increment``` variable

### DISCUSSION THREAD

**Discussion by yrinkel** [astashov/liftosaur#141] [Ideas]
In most of my and other’s programs progression logic, the progressive overload is often set to the minimum increment available (smallest plates X 2 for barbells, smallest available dumbbells and lightest plate on leverage machine/cable stack). 

This is **equipment dependent**, which means we are editing the progression logic every time we change equipment/gym. That can be avoided. 

It would be great to have a ```minimum_increment``` variable available (or any other name if you can think of a better one). Then it’s just a matter of looking up the *lightest* weight available for the currently set equipment and populating the variable with it, and we can happily use it in our scripts.

**Comment by astashov** [comment]
Done!
https://www.reddit.com/r/liftosaur/comments/1lh38zp/added_incrementdecrement_functions/

**Comment by yrinkel** [comment]
Awesome!!
